projects/ngx-mask-lib/src/lib/ngx-mask.service.ts

```diff
@@ -399,9 +399,9 @@ export class NgxMaskService extends NgxMaskApplierService {
         if (!this._renderer || !this._elementRef) {
             return;
         }
-        Promise.resolve().then(
-            () => this._renderer?.setProperty(this._elementRef?.nativeElement, name, value)
-        );
+        Promise.resolve().then(() => {
+            this._renderer?.setProperty(this._elementRef?.nativeElement, name, value);
+        });
```

Comment:
Promise.resolve(this._renderer?.setProperty(this._elementRef?.nativeElement, name, value))

Reply:
postponed